omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -203,7 +255,25 @@ public void setModelAttributes(PageModel model, Account account, OtherAccountDat
 		model.addAttribute("allowedLocales", administrationService.getAllowedLocales());
 		List<ProviderRole> providerRoles = providerManagementService.getAllProviderRoles(false);
 		model.addAttribute("providerRoles", providerRoles);
-		
+
+		List<Extension> customPersonAttributeEditFragments =
+				appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonAttributeFragment");
```

Comment:
Shouldn't the extension name include "Edit"?

Reply:
@dkayiwa  Just changed to "createUser.manageAccountPersonAttributeEditFragment"